app/views/decidim/pages/_standalone.html.erb

```diff
@@ -1,6 +1,7 @@
+
 <div class="wrapper">
   <div class="row column">
-    <div class="text-center">
+    <div class="page-title-wrapper">
```

Comment:
It works perfectly, thanks @Quentinchampenois! 

I just have one feedback, could you please just come back to the original file class, as the titles are now displaying left, and we want them centered as before? 

```suggestion
    <div class="text-center">
```